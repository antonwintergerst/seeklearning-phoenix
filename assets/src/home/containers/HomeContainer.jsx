import React from 'react';

const HomeContainer = () => (
  <div>
    <h1 className="ui header">Get where you want to be in your career</h1>
  </div>
);
export default HomeContainer;
