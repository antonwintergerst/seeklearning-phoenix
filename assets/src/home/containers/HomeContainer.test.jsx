import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import HomeContainer from './HomeContainer';

describe('HomeContainer', () => {
  it('renders as expected', () => {
    const tree = ReactTestRenderer.create(<div><HomeContainer /></div>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
