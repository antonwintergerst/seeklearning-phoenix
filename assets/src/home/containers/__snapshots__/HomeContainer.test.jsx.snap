// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeContainer renders as expected 1`] = `
<div>
  <div>
    <h1
      className="ui header"
    >
      Get where you want to be in your career
    </h1>
  </div>
</div>
`;
